things/database.py

```diff
@@ -685,28 +685,38 @@ def make_filter(column, value):
     }.get(value, default)
 
 
-def make_date_filter(create_var_name, last):
-    """Returns SQL filter to limit date range of the SQL query.
+def make_date_filter(created_column, offset):
+    """
+    Returns SQL filter to limit date range of the SQL query.
+    """
```

Comment:
Please do keep Args and Returns, I like it! My intended suggestion was to rename "Args" into "Parameters" and slightly modify the styling to be consistent with others. See `things.api.tasks` for an example of the numpy/pandas-like docstrings.

Reply:
Oh! Apologies for misinterpreting. I'm glad you liked them. I'll add them back in.